earth_enterprise/src/fusion/gecombineterrain/combineterrain.cpp

```diff
@@ -88,6 +90,7 @@ void TerrainCombiner::CombineTerrainPackets(
 
 void TerrainCombiner::CombineChildren(const TerrainQuadsetGroup &quadset_group,
                                       const QuadtreePath even_path) {
+  BEGIN_PROFILING(combineChildrenProf, "CombineChildren", even_path.AsString());
```

Comment:
When you disable the block, be sure to do it in such a way that it won't execute even_path.AsString()